packaging/spec.in

```diff
@@ -32,6 +38,7 @@ Extensions include:
   - Cluster upgrade wizard
   - Host copy network dialog
   - VM export dialog
+  - VM CPU host pinning dialog
```

Comment:
unrelated?

Reply:
true - I'll leave it for the new release update